packages/translations/src/i18next/i18next.js

```diff
@@ -20,6 +21,24 @@ const ALL_LANGUAGES = Object.freeze({
     ZH_CN: '简体中文',
     ZH_TW: '繁體中文',
 });
+const allowed_languages = { EN: 'English', ID: 'Indonesia', PT: 'Português', ES: 'Español' };
```

Comment:
I think you can move this together to exclude_languages on line 27. We won't need this variable out of the function scope.